Bluepill-cli/Bluepill-cli/Simulator/BPSimulator.m

```diff
@@ -40,7 +41,7 @@ + (instancetype)simulatorWithConfiguration:(BPConfiguration *)config {
 }
 
 - (NSMutableDictionary *)createSimulatorAndInstallAppWithBundles:(NSArray<BPXCTestFile *>*)testBundles {
```

Comment:
I would break out this method or at least rename it. It's called `createSimulatorAndInstallAppWithBundles` but effectively it also launches the app with the injected code to get the tests.